src/main/java/mate/academy/springboot/swagger/model/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.swagger.model.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data

```

Comment:
I think you don't need all methods this annotation provide, maybe it would be enoph to have only setters and getters